Irem ozcelik

irem-ozcelik opened the pull request:
required changes are added

irem-ozcelik committed:
Update Program.cs

irem-ozcelik committed:
Update Program.cs

irem-ozcelik committed:
unused endpoints are removed

LinusTyr reviewed (changes requested):
This doesn't work.

I don't think you've done any testing at all.

There are two major problems with running this code.

As soon as you attempt to run it you will see what that is.

irem-ozcelik committed:
Update Program.cs

irem-ozcelik committed:
Update Program.cs

iremozc commented:
I have made changes so the code runs hopefully

LinusTyr reviewed (changes requested):
First off the application still doesn't work.

Any time you are developing anything (it could be making food) you need to ensure that you reach your expected outcome. With cooking it is about tasting your food, in programming you need to test.

Use a tool so that you can see if you get things to work.

If you use JetBrains Rider you can use this AdditionApi.http file :

```
@AdditionApi_HostAddress = http://localhost:5262

PUT {{AdditionApi_HostAddress}}/localStorage/setItem/key1
Content-Type: application/json

{
  "value": "exampleValue"
}

###

GET {{AdditionApi_HostAddress}}/localStorage/getItem/key1
Accept: application/json

###
```
<img width="897" height="586" alt="image" src="https://github.com/user-attachments/assets/8ae2c5f3-21dc-42f0-b8ce-ecfb4294fb47" />


otherwise you can use postman to send Http Requests to the API.

Once you have ensured that your API runs as exectedly. Also remove all commented out code (bad practice to leave it there) as well as remove all unused files (Weather../Order..).

LinusTyr commented:
Thank you for your changes.

We still have some ways to go. Hang in there.

Take a look at my comments in your pull request.

Get back to me if you have any questions.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virusfri.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Den fre 31 okt. 2025 kl 10:01 skrev iremozc ***@***.***>:

> *iremozc* left a comment
> (SFX-IT-Orienteringskursen-VT25/assign3-addition-api#24)
> <https://github.com/SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24#issuecomment-3471984971>
>
> I have made changes so the code runs hopefully
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24#issuecomment-3471984971>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQTOF24QI62CO63NTFPI35332MQNTAVCNFSM6AAAAACJO267TWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTINZRHE4DIOJXGE>
> .
> You are receiving this because you commented.Message ID:
> <SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24/c3471984971@
> github.com>
>

irem-ozcelik committed:
Replace weather and order endpoints with localStorage API

Removed unused files and their related code and commented lines. Corrected endpoints to fix errors. Updated HTTP requests

iremozc commented:
Hey Linus! I have tested the code now and got these response codes from the console:
PUT http://localhost:5262/localStorage/setItem/key1

HTTP/1.1 204 No Content
Date: Mon, 03 Nov 2025 14:01:05 GMT
Server: Kestrel

<Response body is empty>

Response code: 204 (No Content); Time: 66ms (66 ms); Content length: 0 bytes (0 B)
Thank you for your guidance!

LinusTyr reviewed (approved):
Good job!

LinusTyr commented:
Great that you got it working!

It's all about them short feedback loops ;).

Den mån 3 nov. 2025 kl 15:07 skrev iremozc ***@***.***>:

> *iremozc* left a comment
> (SFX-IT-Orienteringskursen-VT25/assign3-addition-api#24)
> <https://github.com/SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24#issuecomment-3480733408>
>
> Hey Linus! I have tested the code now and got these response codes from
> the console:
> PUT http://localhost:5262/localStorage/setItem/key1
>
> HTTP/1.1 204 No Content
> Date: Mon, 03 Nov 2025 14:01:05 GMT
> Server: Kestrel
>
> Response code: 204 (No Content); Time: 66ms (66 ms); Content length: 0
> bytes (0 B)
> Thank you for your guidance!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24#issuecomment-3480733408>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQTOF23MGC65Y2V4YIMCWUD325ORLAVCNFSM6AAAAACJO267TWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTIOBQG4ZTGNBQHA>
> .
> You are receiving this because you commented.Message ID:
> <SFX-IT-Orienteringskursen-VT25/assign3-addition-api/pull/24/c3480733408@
> github.com>
>
